packages/cli/test/mocks/project.ts

```diff
@@ -206,6 +213,43 @@ export function useProject(project: Partial<Project> = defaultProject) {
     Object.assign(project, req.body);
     res.json(project);
   });
+  client.scenario.get(
+    `/v1/env/pull/${project.id}/:target?/:gitBranch?`,
```

Comment:
trailing `?` necessary?

Reply:
yes, without it the tests don't pass when you don't provide a git branch - it marks both target & gitbranch paths as optional, and we have the same configuration in routes.ts on API